chore: redirect anbox-cloud.io to canonical.com/anbox-cloud

muhammad-ali-pk opened the pull request:
## Done

- Added a redirect in permanent redirects

## QA

- View the site in your web browser at: https://anbox-cloud-io-376.demos.haus/
- Verify the root is redirected to canonical.com/anbox-cloud

## Issue / Card

Fixes #[WD-22302](https://warthogs.atlassian.net/browse/WD-22302)


[WD-22302]: https://warthogs.atlassian.net/browse/WD-22302?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

muhammad-ali-pk committed:
redirect all requests to canonical.com/anbox-cloud

webteam-app commented:
### [<img src='https://assets.ubuntu.com/v1/6baef514-ubuntu-circle-of-friends-large.svg' height=32 width=32> Demo</img>](https://anbox-cloud-io-376.demos.haus)
### [<img src='https://assets.ubuntu.com/v1/e512b0e2-jenkins.svg' height=32 width=32> Jenkins </img>](https://jenkins.canonical.com/webteam/job/start-demo/3392)
### [<img src='https://assets.ubuntu.com/v1/7144ec6d-logo-jaas-icon.svg' height=32 width=32> demos.haus </img>](https://demos.haus)

muhammad-ali-pk committed:
Fix test

petesfrench commented:
There are some other template paths that can still be reached, such as https://anbox-cloud-io-376.demos.haus/contact-us

muhammad-ali-pk committed:
Redirect all static links

petesfrench reviewed (approved):
LGTM

muhammad-ali-pk commented:
@petesfrench Thanks. Added redirects for other static links as well.
PS., I added the root redirect as a backup in the meantime the IS adds a global redirect for anbox-cloud.io domain which should have redirected all requests. Since this repo had only a few static links, it was easier to add redirect per link basis. What would the procedure have been for a bigger repo? Adding all redirects would have been impractical, and should have been handled by the IS redirect?

petesfrench commented:
@muhammad-ali-pk You can do `(?P<path>.*): https://canonical.com/anbox-cloud`
